lib/index.js

```diff
@@ -91,21 +101,48 @@ EslintValidationFilter.prototype.constructor = EslintValidationFilter;
 EslintValidationFilter.prototype.extensions = ['js'];
 EslintValidationFilter.prototype.targetExtension = 'js';
 
-EslintValidationFilter.prototype.processString = function processString(content, relativePath) {
-  'use strict'; // eslint-disable-line strict
+EslintValidationFilter.prototype.baseDir = function baseDir() {
+  return process.cwd();
+};
+
+EslintValidationFilter.prototype.cacheKeyProcessString = function cacheKeyProcessString(content, relativePath) {
+  return md5Hex([content, relativePath, JSON.stringify(this.options)]);
```

Comment:
We need to use json-stable-stringify as the unordered json stringify busts the cache incorrectly